bin/lib/release.js

```diff
@@ -56,6 +57,10 @@ export default (releaseType) => {
         throw new Error('No .gitignore file');
       }
 
+      let outputFile = path.resolve(__dirname, `../../styles/${OUTPUT_CSS_FILENAME}`);
+      console.log(`- Deleting output file ${outputFile}`.green);
+      yield pUnlink(outputFile);
```

Comment:
Yeah, sadly fs.exists returns a `bool` not `err, bool` so im not 100% sure what happens when you `denodeify` it see: https://nodejs.org/api/fs.html#fs_fs_exists_path_callback so I guess we have to use it :(

:+1:  for code !